src/pages/public_registration/LogPanel.tsx

```diff
@@ -120,20 +126,58 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
               </Typography>
             )}
           </Box>
-          <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
+          <Tooltip title={new Date(registration.createdDate).toLocaleTimeString()} enterDelay={tooltipDelay}>
+            <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
+          </Tooltip>
         </StyledStatusMessageBox>
       )}
       {logs
         .sort((a, b) => new Date(a.modifiedDate).getTime() - new Date(b.modifiedDate).getTime())
         .map((logItem, index) => {
           const bgColor = logItem.type === 'PublishingRequest' ? 'publishingRequest.main' : 'doiRequest.main';
+          const modifiedDate = new Date(logItem.modifiedDate);
           return (
             <StyledStatusMessageBox key={index} sx={{ bgcolor: `${bgColor}` }}>
               <Typography>{logItem.description}</Typography>
-              <Typography>{new Date(logItem.modifiedDate).toLocaleDateString()}</Typography>
+              <Tooltip title={modifiedDate.toLocaleTimeString()} enterDelay={tooltipDelay}>
+                <Typography>{modifiedDate.toLocaleDateString()}</Typography>
+              </Tooltip>
+              {logItem.descriptionAppendage && logItem.descriptionAppendage?.length > 0 && (
+                <Box
+                  component="ul"
+                  sx={{
+                    gridColumn: '1/3',
+                    m: '0',
+                    p: 'inherit',
+                    color: 'black',
+                  }}>
+                  {logItem.descriptionAppendage.map((desc, index) => (
+                    <li key={index}>
+                      <Tooltip title={desc === t('common.deleted') ? '' : desc} enterDelay={tooltipDelay}>
+                        <Typography
+                          sx={{
+                            whiteSpace: 'nowrap',
+                            overflow: 'hidden',
+                            textOverflow: 'ellipsis',
+                          }}>
+                          {desc}
+                        </Typography>
+                      </Tooltip>
+                    </li>
+                  ))}
+                </Box>
+              )}
             </StyledStatusMessageBox>
           );
         })}
     </Box>
   );
 };
+
+function getFileNamesForTicket(ticket: PublishingTicket, registration: Registration, t: TFunction<'translation'>) {
+  return ticket.approvedFiles.map(
+    (identifier) =>
+      getAssociatedFiles(registration.associatedArtifacts).find((file) => file.identifier === identifier)?.name ??
+      t('common.deleted')
```

Comment:
Er det tilstrekkelig å returnere "Slettet" for slettede filer? Er jo en teoretisk corner case at filen faktisk heter "Slettet" (uten filending)? Burde man legge på italic tekststil eller noe på slettede innslag?

Reply:
Enig i at kursiv hadde vært bra! Enten at slettet er i kursiv, eller alle fil-filnavn.
Kanskje mer utfyllende med "Slettet fil", eller "Fil fjernet senere" elns også? kan ta en runde med designer rundt det i morgen.

Når jeg åpnet resultatet ditt med mange filer som er slettet ser det egentlig ut som en bug, siden jeg ikke ser hva som er slettet, men det er jo en ganske ekstrem case som ikke skal skje i realiteten heller da, så mulig det ikke er noe problem 😅